onadata/apps/logger/management/commands/transfer_project.py

```diff
@@ -0,0 +1,93 @@
+"""Functionality to transfer a project form one owner to another."""
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from onadata.apps.logger.models import Project, XForm, DataView, MergedXForm
+from onadata.apps.logger.models.project import set_object_permissions \
+    as set_project_permissions
+from onadata.libs.utils.project_utils import set_project_perms_to_xform
+
+
+class Command(BaseCommand):  # pylint: disable=C0111
+    help = 'A command to reassign a project form one user to the other.'
```

Comment:
This command transfers all projects from one user to another user, the description seems to indicate we are only transferring one project.

Could we have two options:
1. `--project` -  the project id of the project to transfer
2. `--all` - transfer all projects